src/OSPSuite.Core/Domain/SensitivityAnalyses/SensitivityAnalysis.cs

```diff
@@ -73,22 +70,6 @@ public virtual void RemoveSensitivityParameter(SensitivityParameter sensitivityP
          HasChanged = true;
       }
 
-      public virtual void AddDynamicPKParameter(DynamicPKParameter dynamicPKParameter)
```

Comment:
UserDefinedPKPArameter are now added globally and not per SA.. Much more ergnomic